apps/drupal/templates/layout/region--footer.html.twig

```diff
@@ -13,17 +13,13 @@
  */
```

Comment:
@jryanconklin: we don't need a footer region template. If there are wrapper classes, place them up in page.